database/queries.js

```diff
@@ -17,4 +17,4 @@ const getUsers = (req, res) => {
   });
 };
 
-module.exports = { getUsers };
```

Comment:
Most of this looks great! Happy to see you've finished your schema and seeded your database.  Can't wait to see how you move forward with your 'Hello World'.